mdal/frmts/mdal_ply.cpp

```diff
@@ -577,15 +580,44 @@ void MDAL::DriverPly::save( const std::string &fileName, const std::string &mesh
     }
   }
 
+  libply::Metadata meta;
+
+  meta.emplace( "crs", mesh->crs() );
+  MDAL::Metadata metadata = mesh->metadata;
+  libply::File::Format format = libply::File::Format::BINARY_LITTLE_ENDIAN;
+
+  const std::unordered_map<std::string, libply::File::Format> FORMAT_MAP =
```

Comment:
any reasons to be upper case?

Reply:
Nope. That is a brain_fart! I think it was an unconscious association with the line above :(